notebooks/building_production_ml_systems/solutions/1_training_at_scale_vertex.ipynb

```diff
@@ -8,7 +8,8 @@
     "**Learning Objectives:**\n",
```

Comment:
<p>Can you please add a header here saying it's a "Note" or "Optional" so its clear that it's just an FYI and not something the students need to do?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>